Assets/Scripts/MainMenu.cs

```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    public void PlayGame()
+    {
+        SceneManager.LoadScene("Level1"); //temp change to level selection later
```

Comment:
what I don't like in Unity is when there's a bunch of one line scripts laying around, but sadly I also don't know how to avoid that :/

Reply:
:<